google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/MavenBuildDeploymentSource.java

```diff
@@ -130,17 +122,12 @@ public File getFile() {
     // The environment will be null for newly deserialized deployment sources to ensure freshness.
     // In this case, we need to try and reload the environment.
     if (environment == null) {
-      Optional<AppEngineEnvironment> appEngineEnvironment =
-          projectService.getModuleAppEngineEnvironment(getModule());
-
-      appEngineEnvironment.ifPresent(
-          env -> {
-            environment = env;
-            if (env.isFlexible()) {
-              targetBuild.append(".");
-              targetBuild.append(mavenProject.getPackaging());
-            }
-          });
+      projectService.getModuleAppEngineEnvironment(getModule()).ifPresent(env -> environment = env);
+    }
+
+    if (environment != null && environment.isFlexible()) {
+      targetBuild.append(".");
```

Comment:
let's verify that standard deployment of maven artifacts is working.

Reply:
I just checked and it does work for App Engine standard deployments. (Offline I mentioned that it doesn't work, but that was because my Maven project was not configured to build a WAR, so the exploded folder wasn't created.)